src/router.js

```diff
@@ -1,8 +1,9 @@
 const http = require('http');
-const users = require('./static.js');
+const users = require('./handlers/static.js');
```

Comment:
Not sure why this file has been moved to a `handlers` directory when it's clearly not a handler.